lxd/images.go

```diff
@@ -2352,7 +2499,8 @@ func imageRefresh(d *Daemon, r *http.Request) response.Response {
 
 	// Begin background operation
 	run := func(op *operations.Operation) error {
-		return autoUpdateImage(d, op, imageId, imageInfo, project)
```

Comment:
@monstermunchkin is `imageRefresh` even needed anymore?

Reply:
@tomponline No. This is for testing only.

Reply:
So it can be removed? I couldnt see references to it in the code base.

Reply:
It's used in our test suite to trigger an image refresh.

Reply:
@monstermunchkin ah ok, yes I just found it, was searching for `imageRefresh(` but its used as a handler in `imageRefreshCmd`